todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,29 @@
+import pymongo
+from flask import Flask
+from todo_app.flask_config import Config
+from todo_app.item import Item
+
+def get_items( app ):
+    items = []
+    client = pymongo.MongoClient(app.config['MONGO_CONNECTION_STRING'])
+    db = client[app.config['MONGO_DATABASE']]
+    collection = db['Items']
+    for mongo_item in collection.find():
+        items.append (Item.from_mongo_item(mongo_item))
+    return items
+
+def add_item(app, title):
+    client = pymongo.MongoClient(app.config['MONGO_CONNECTION_STRING'])
+    db = client[app.config['MONGO_DATABASE']]
+    collection = db['Items']
+    newItem = {"title": title, "status": "To Do"}
+    collection.insert_one(newItem)
+
+def complete_item(app, title):
+    client = pymongo.MongoClient(app.config['MONGO_CONNECTION_STRING'])
+    db = client[app.config['MONGO_DATABASE']]
+    collection = db['Items']
+    myquery = { "title": title, "status": "To Do"}
```

Comment:
I'm always a little wary of querying against a title - what if you had two to-dos with the same name and just wanted to update one of them? Possibly nicer would be to use their ids - though we would have to be careful with types
```
from bson import ObjectId

...

    myquery = { "_id": ObjectId(id) }
```

Reply:
Obviously if we were to do this we'd need to tweak the other parts of the code back to make sure it's the id that gets fed through here not the title